### PR TITLE
Add required information in the CSV to show in the UI

### PR DESCRIPTION
**Pull request by iamniting**
Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

### CODE REVIEW

File: config/manifests/bases/odf-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -3,8 +3,13 @@ kind: ClusterServiceVersion
 metadata:
   annotations:
     alm-examples: '[]'
-    capabilities: Basic Install
+    capabilities: Deep Insights
```
Comment (jarrpa) [inline config/manifests/bases/odf-operator.clusterserviceversion.yaml:6]: ...do we actually qualify for Deep insights? I'm pretty sure ocs-operator doesn't, and I don't remember seeing that odf-operator exposes enough functionality for it either.

File: config/manifests/bases/odf-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -3,8 +3,13 @@ kind: ClusterServiceVersion
...
+    categories: Storage
+    containerImage: quay.io/ocs-dev/odf-operator:latest
+    description: ODF operator provides a way to deploy OCS and IBM flashsystem cluster
```
Comment (jarrpa) [inline config/manifests/bases/odf-operator.clusterserviceversion.yaml:9]:
```suggestion
    description: OpenShift Data Foundation provides a common control plane for storage solutions on OpenShift Container Platform.
```

**Review by jarrpa — CHANGES_REQUESTED** [review]
Oh yes, we definitely need something like this. Some additional comments:

* Can you add a `labels` section to the metadata, as we have in ocs-operator? odf-operator by definition should support all the architectures ocs-operator does.

File: config/manifests/bases/odf-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (iamniting) [reply config/manifests/bases/odf-operator.clusterserviceversion.yaml:6]: I copied it from the ocs-operator itself

File: config/manifests/bases/odf-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (iamniting) [reply config/manifests/bases/odf-operator.clusterserviceversion.yaml]: done

File: config/manifests/bases/odf-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -16,11 +21,12 @@ spec:
       kind: StorageSystem
       name: storagesystems.odf.openshift.io
       version: v1alpha1
-  description: OpenShift Data Foundation
+  description: |
+    **Red Hat OpenShift Data Foundation** deploys four operators.
```
[the remark this answers is not in the document]
Comment (iamniting) [reply config/manifests/bases/odf-operator.clusterserviceversion.yaml:26]: done

**Comment by iamniting** [comment]
> * Can you add a `labels` section to the metadata, as we have in ocs-operator? odf-operator by definition should support all the architectures ocs-operator does.

done

File: config/manifests/bases/odf-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (jarrpa) [reply config/manifests/bases/odf-operator.clusterserviceversion.yaml:6]: Huh... okay! We'll see if anyone brings it up again. :laughing:

**Review by jarrpa — APPROVED** [review]
/lgtm

**Pull request merged**